capella_model_explorer/app.py

```diff
@@ -222,7 +222,33 @@ def template_page(template_id: str, model_element_uuid: str = "") -> t.Any:
                     selected_model_element_uuid=model_element_uuid,
                 ),
                 id="template-sidebar",
-                cls="fixed min-w-96 w-96 p-4 print:hidden",
+                cls=(
+                    "aria-expanded:min-w-full",
+                    "aria-expanded:opacity-100",
+                    "aria-expanded:w-full",
+                    "duration-500",
+                    "fixed",
+                    "lg:min-w-96",
+                    "lg:opacity-100",
+                    "lg:w-96",
+                    "opacity-0",
+                    "p-4",
+                    "print:hidden",
+                    "transition-with",
+                    "w-0",
```

Comment:
The animations on the sidebar look a bit clunky.

- Opening the sidebar fades it in, while closing it transitions the width to zero while fading out - we should use the same animation for both cases.

- The width transition creates a "squishing together" effect, which looks quite awkward, as the contained model element buttons grow vertically before eventually disappearing. For a proper sliding effect, we should keep the width constant and transition the horizontal position instead.